tests/ui/job-view/bugfiler_test.jsx

```diff
@@ -187,73 +187,72 @@ describe('BugFiler', () => {
       ' | uncaught exception - TypeError: this.textbox.popup.oneOffButtons is undefined at ' +
       'searchbar_XBL_Constructor@chrome://browser/content/search/search.xml:95:9';
     const expected =
-      'accessible/tests/mochitest/states/test_expandable.xul | uncaught exception - TypeError: this.textbox.popup.oneOffButtons is undefined at ' +
-      'searchbar_XBL_Constructor@chrome://browser/content/search/search.xml:95:9';
+      'Intermittent accessible/tests/mochitest/states/test_expandable.xul | uncaught exception - TypeError: this.textbox.popup.oneOffButtons is undefined at searchbar_XBL_Constructor@chrome://browser/content/search/search.xml:95:9';
     const displayed = SummaryAndExpected(rawSummary);
-    expect(displayed).toBeInTheDocument(expected);
+    expect(displayed).toHaveValue(expected);
   });
 
   test('should parse xpcshell summaries', () => {
     const rawSummary =
       'xpcshell-child-process.ini:dom/indexedDB/test/unit/test_rename_objectStore_errors.js | application crashed [@ mozalloc_abort(char const*)]';
     const expected =
-      'dom/indexedDB/test/unit/test_rename_objectStore_errors.js | application crashed [@ mozalloc_abort(char const*)]';
```

Comment:
there are cases now where the crash is at the beginning of the line and the test name is at the end.  I am sure I had some test cases for that, but maybe not in bugfiler